docs/installation/index.md

```diff
@@ -5,10 +5,24 @@ sidebar: sidebars/documentation.html
 script: osmenu.js
 ---
 
+{% if site.version == null %}
+# Katello Nightly Installation
+{% else %}
 # Katello Installation
+{% endif %}
+
+{% if site.version == null %}
+  <div class="alert alert-danger">
+    These are the instructions for installing the unstable nightly release of katello!
+  </div>
+{% elsif site.version != site.latest %}
+  <div class="alert alert-danger">
+    These instructions are for installing an older version of Katello, see <a href="/docs/{{ site.latest }}/installation/index.html">{{ site.latest }}</a> for the latest stable version.
```

Comment:
This will look odd with RC's, a user might be looking at 2.4 RC and we're telling them it's old and they should look at 2.3 :-)

What do you think of something like `These instructions are for installing Katello {{ site.version }}, but the latest stable is {{ site.latest }}`?